components/playlist/renderer/playlist_js_handler.h

```diff
@@ -22,11 +24,17 @@ namespace playlist {
 
 class PlaylistJSHandler {
  public:
-  explicit PlaylistJSHandler(content::RenderFrame* render_frame);
+  PlaylistJSHandler(content::RenderFrame* render_frame,
+                    const int32_t isolated_world_id);
   ~PlaylistJSHandler();
 
   void AddWorkerObjectToFrame(v8::Local<v8::Context> context);
 
+  void SetDetectorScript(const blink::WebString& script);
+  void allow_to_run_script_on_main_world() {
```

Comment:
Casing

Reply:
If you're meaning that it should be `AllowToRunScriptOnMainWorld()`, I believe using `snake_case` for `inline` method is the convention.